README.md

```diff
@@ -21,3 +21,7 @@ Use command line (Cake build):
 to get the right results, or exclude the tests from mock-assembly. 
 
 For more details see https://github.com/nunit/docs/wiki/Packaging-the-V3-Adapter  
+
+
+## Announcement
+From version 3.9 the NUnit3Testadapter will stop supporting Visual Studio 2012 RTM (!), note only RTM, the later updates will still be supported fully. If you're using VS 2012 and want to update NUnit3TestAdapter , please update your Visual Studio RTM to any of the subsequent updates (Update 1-5)
```

Comment:
`NUnit3Testadapter` should be `NUnit3TestAdapter` to match the second occurrence and there's a spurious space before the comma.